openfecli/commands/gather.py

```diff
@@ -4,12 +4,13 @@
 import click
 import os
 import pathlib
+import sys
 from typing import Callable, Literal, List
-import warnings
 
 from openfecli import OFECommandPlugin
 from openfecli.clicktypes import HyphenAwareChoice
 
+FAIL_STR = "NaN" # string used to indicate a failed run in output tables.
```

Comment:
I think we should use 
```suggestion
FAIL_STR = "Err" # string used to indicate a failed run in output tables.
```
since "NaN" (IMHO) implies some sort of numerical issue, but here we are just saying "there is an error here"

Reply:
That's a good point, I'm not sure which would be better :/ However, if we do go for "Err", given the width of the field, should we just got for the full "Error"?

Reply:
I agree and switched it to `Error`